src/opera/util/run_utils.py

```diff
@@ -131,9 +135,9 @@ def create_sas_command_line(sas_program_path, sas_runconfig_path,
             if not os.access(executable_path, mode=os.X_OK):
                 raise OSError(f"Requested SAS program path {sas_program_path} exists, "
                               f"but does not have execute permissions.")
-        # Otherwise, sas_program_path might be a python module path
         else:
-            command_line = ['python3', '-m', sas_program_path]
+            error_msg = "Configured executable from the RunConfig could not be found"
+            logger.critical(pge_name, ErrorCode.EXECUTABLE_FROM_RUNCONFIG_NOT_FOUND, error_msg)
```

Comment:
Rather than make the logger.critical call within this function (which necessitates passing a logger in as an additional argument) I think I would rather have this function raise an `OSError` for this case.

That way, the callers of the function (the PGE classes) will catch the OSError, then use their own logger instance to make the `critical` call.

Reply:
Yeah, I saw the back to back errors reporting.